docs/source/getting-started/registering-first-model/step3-load-model.rst

```diff
@@ -0,0 +1,113 @@
+load a model from the model registry via a specific model URI and perform inference.
+
+.. code-block:: python
+
+    import mlflow.sklearn
+    from sklearn.datasets import make_regression
+
+    model_name = "sk-learn-random-forest-reg-model"
+    model_version = "latest"
+
+    # Load the model from the Model Registry
+    model_uri = f"models:/{model_name}/{model_version}"
+    model = mlflow.sklearn.load_model(model_uri)
+
+    # Generate a new dataset for prediction and predict
+    X_new, _ = make_regression(n_features=4, n_informative=2, random_state=0, shuffle=False)
+    y_pred_new = model.predict(X_new)
+
+    print(y_pred_new)
+
+Note that if you're not using sklearn, if your model flavor is supported, you should use the 
+specific model flavor load method e.g. ``mlflow.<flavor>.load_model()``. If the model flavor is 
+not supported, you should leverage :py:func:`mlflow.pyfunc.load_model()`. Throughout this tutorial
+we leverage sklearn  for demonstration purposes.
+
+
+Example 0: Load via Tracking Server
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+A model URI is a unique identifier for a serialized model. Given the model artifact is stored with
+experiments in the tracking server, you can use the below model URIs to bypass the model registry
+and load the artifact into memory.
+
+1. **Absolute local path**: ``mlflow.sklearn.load_model("/Users/me/path/to/local/model")``
+2. **Relative local path**: ``mlflow.sklearn.load_model("relative/path/to/local/model")``
+3. **Run id**: ``mlflow.sklearn.load_model(f"runs:/{mlflow_run_id}/{run_relative_path_to_model}")``
+
+However, unless you're in the same environment that you logged the model, you typically won't have
+the above information. Instead, you should load the model by leveraging the model's name and 
+version.
+
+Example 1: Load via Name and Version 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+To load a model into memory via the ``model_name`` and monotonically increasing ``model_version``,
+use the below method:
+
+.. code-block:: python
+
+    model = mlflow.sklearn.load_model(f"models:/{model_name}/{model_version}")
+
+While this method is quick and easy, the monotonically increasing model version lacks flexibility. 
+Often, it's more efficient to leverage a model version alias.
+
+Example 2: Load via Model Version Alias
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Model version aliases are user-defined identifiers for a model version. Given they're mutable after
+model registration, they decouple model versions from the code that uses them.
+
+For instance, let's say we have a model version alias called ``production_model``, corresponding to 
+a production model. When our team builds a better model that is ready for deployment, we don't have 
+to change our serving workload code. Instead, in MLflow we simply delete the ``production_model`` 
+alias from the old model version and create a new ``production_model`` version alias for the new 
+model version. It's that simple!
```

Comment:
Actually, what you do is you reassign the production alias from the old model version to the new one. This can be done simply in the UI. In the API, you can just run `client.set_registered_model_alias` with the same model name, alias name, and new model version ID.

Reply:
Great catch. Resolved